R/dataset.R

```diff
@@ -110,7 +110,7 @@ Dataset.template <- function(id, env = solvebio:::.solveEnv) {
 #' \url{https://docs.solvebio.com/}
 #'
 #' @export
-Dataset.data <- function(id, filters, row.names = NULL, env = solvebio:::.solveEnv, ...) {
+Dataset.data <- function(id, filters, col.names = NULL, env = solvebio:::.solveEnv, ...) {
```

Comment:
lets change col.names here to `fields` since it isn't just the name, it is the fields response.

Reply:
Would still use `fields` here since it is a dataframe of the field objects.